GenumKit/Parsers/StoryboardParser.swift

```diff
@@ -86,7 +86,7 @@ public final class StoryboardParser {
     }
   }
 
-  public func addStoryboardAtPath(_ path: String) {
+  public func addStoryboard(atPath path: String) {
```

Comment:
To be consistent with your naming in fonts & color parsers, I'd change this to addStoryboard(at path: )